packages/styles/src/components/button-group.scss

```diff
@@ -17,7 +17,9 @@
 
     // added a transition delay on the "z-index" so the button only moves back after all other transitions
     z-index: 0;
-    transition: z-index 0s ease (0.5 * button.$btn-transition-duration), button.$btn-transition;
+    transition:
+      z-index 0s ease (0.5 * button.$btn-transition-duration),
+      button.$btn-transition;
```

Comment:
Line break rules has been fixed: https://prettier.io/blog/2023/07/05/3.0.0.html#fix-line-break-in-css-declaration-with-comma-14208httpsgithubcomprettierprettierpull14208-by-mvorisekhttpsgithubcommvorisek